PYME/experimental/triangle_mesh_utils.c

```diff
@@ -58,7 +58,7 @@ DB: Performance suggestions:
 
 */
 
-static void update_single_vertex_neighbours(int v_idx, halfedge_t *halfedges, void *vertices_, void *faces_)
+static void update_single_vertex_neighbours(int32_t v_idx, halfedge_t *halfedges, void *vertices_, void *faces_)
 {
     int32_t i, k, orig_idx, curr_idx, twin_idx, tmp;
```

Comment:
`i`, an `k` should definitely be `int`, rather than `int32_t`. Probably 6 of one/half a dozen of the other for the other variables.

Reply:
They were all `int32_t` in the version of meshing that last behaved what I think is correctly, so I changed them back to make sure this wasn't what was causing the issue.